src/interface/desktop/renderer.js

```diff
@@ -61,6 +61,13 @@ toggleFoldersButton.addEventListener('click', () => {
 function makeFileElement(file) {
     let fileElement = document.createElement("div");
     fileElement.classList.add("file-element");
+
+    let fileIconElement = document.createElement("img");
+    fileIconElement.classList.add("card-icon");
+    fileIconElement.src = "./assets/icons/plaintext.svg";
```

Comment:
Hm, I don't think it makes sense to include the icons here. This will make the UX very redundant, since the section header already has this icon and says `Files`. I'd say drop the icon for now. We could use some kind of `sync` indicator here eventually?

Reply:
Makes sense, I'll revert the change. The main thing I wanted was to make the files, folders indexed to not seem like editable fields (as they're not). I've done something to make it look different from the other elements, hopefully it can reduce chance of confusion